pkgs/applications/editors/notepad-next/default.nix

```diff
@@ -0,0 +1,36 @@
+{ mkDerivation, lib, fetchFromGitHub, qmake, libsForQt5 }:
+
+mkDerivation rec {
+  pname = "notepad-next";
+  version = "0.5.1";
+
+  src = fetchFromGitHub {
+    owner = "dail8859";
+    repo = "NotepadNext";
+    rev = "v${version}";
+    sha256 = "sha256-J7Ngt6YtAAZsza2lN0d1lX3T8gNJHp60sCwwaLMGBHQ=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ qmake libsForQt5.qt5.qttools ];
+  qmakeFlags = [ "src/NotepadNext.pro" ];
+
+  # References
+  #  https://github.com/dail8859/NotepadNext/blob/master/doc/Building.md
+  #  https://github.com/dail8859/NotepadNext/issues/99
+  postPatch = ''
+    substituteInPlace ./src/NotepadNext/NotepadNext.pro --replace /usr $out
```

Comment:
Do you meant to link https://github.com/dail8859/NotepadNext/pull/124?

Reply:
I refereed more to https://github.com/dail8859/NotepadNext/issues/99#issuecomment-1100761965 but I can change it to this as well as mentioned in https://github.com/dail8859/NotepadNext/issues/126#issuecomment-1120403377 we should not need it in the future when a release with this change is available - even I don't already know how to use the new prefix option.

Reply:
The qmake setup hook should set the prefix by default: https://github.com/NixOS/nixpkgs/blob/634141959076a8ab69ca2cca0f266852256d79ee/pkgs/development/libraries/qt-5/hooks/qmake-hook.sh#L10